src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -705,3 +704,77 @@ def test_storage_account_update_delete_retention_policy(self, resource_group, st
         result = self.cmd('{cmd} --enable-delete-retention false -n {sa} -g {rg}').get_output_in_json()
         self.assertEqual(result['deleteRetentionPolicy']['enabled'], False)
         self.assertEqual(result['deleteRetentionPolicy']['days'], None)
+
+
+class StorageAccountPrivateLinkScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_sa_plr')
+    @StorageAccountPreparer(name_prefix='saplr', kind='StorageV2', sku='Standard_LRS')
+    def test_storage_account_private_link(self, storage_account):
+        self.kwargs.update({
+            'sa': storage_account
+        })
+        self.cmd('storage account private-link-resource list --account-name {sa} -g {rg}', checks=[
+            self.check('length(@)', 6)])
```

Comment:
Should we support `az storage account private-link-resource list --id {}`?

Reply:
done

Reply:
> good point. I just enabled id_part for acocunt_name and it supports --ids now.

Great.